src/Esprima/JavascriptParser.cs

```diff
@@ -3451,7 +3451,7 @@ private List<StatementListItem> ParseDirectivePrologues()
 
         // https://tc39.github.io/ecma262/#sec-method-definitions
 
-        private bool QualifiedPropertyName(Token token)
+        private static bool QualifiedPropertyName(Token token)
         {
             switch (token.Type)
```

Comment:
mental not for me, need to check how this inlines as uses `switch` instead of `if`s, also probably could be simpler (and probably less obvious) via `<` `>` operators